### PR TITLE
[doc] Update hello_world.md

### PR DESCRIPTION
**Pull request by vai9er**
Issue: #

### CODE REVIEW

File: docs/lang/articles/get-started/hello_world.md [hunk 1]
```diff
@@ -70,44 +71,57 @@ Save the code above to your local machine and run this program.
...
 
-### Define a field
+```python{4}
+import taichi as ti
+import taichi.math as tm
+
+ti.init(arch=ti.gpu)
+```
+
+### Define a Field
```
Comment (Olinaaaloompa) [inline docs/lang/articles/get-started/hello_world.md:111]:
```suggestion
### Define a Taichi field
```

File: docs/lang/articles/get-started/hello_world.md [hunk 2]
```diff
@@ -70,44 +71,57 @@ Save the code above to your local machine and run this program.
...
+
+:::note
+`ti.field(x,y)` - Defines a Field whose shape is of `y` and whose elements are of type `x`.
 
-Let's move on to the next two lines.
+*Field* is the most important and frequently used data structure in Taichi. You can compare it to NumPy's `ndarray` or PyTorch's `tensor`. But Taichi's Field is more flexible. For example, a Taichi Field can be [spatially sparse](../basic/sparse.md) and easily [switched between different data layouts](../basic/layout.md).
+
+We will introduce more advanced features of field in other scenario-based tutorials. For now, it suffices to know that the Field `pixels` is a dense 2D array.
```
Comment (Olinaaaloompa) [inline docs/lang/articles/get-started/hello_world.md:118]:
```suggestion
We will introduce more advanced features of fields in other scenario-based tutorials. For now, it suffices to know that the field `pixels` is a dense 2D array.
```

**Review by Olinaaaloompa — CHANGES_REQUESTED** [review]
Re-checked the capitalization of `field`. Everything else is cool!

**Review by Olinaaaloompa — APPROVED** [review]
LGTM! Thanks!

File: docs/lang/articles/get-started/hello_world.md [hunk 3]
```diff
@@ -70,44 +71,57 @@ Save the code above to your local machine and run this program.
...
 
-```python
-ti.init(arch=ti.gpu)
-```
 
-This line calls `ti.init()` to customize your Taichi runtime depending on the arguments passed into it. For now, we only introduce the most important argument, namely, `arch`.
+### Initialize Taichi
+
+:::note
+`ti.init(x:Optional)`- Initializes Taichi environment and allows you to customize your Taichi runtime depending on the `x` number of arguements passed into it.
```
Comment (neozhaoliang) [inline docs/lang/articles/get-started/hello_world.md:95]:
```suggestion
`ti.init(**kwargs)`- Initializes Taichi environment and allows you to customize your Taichi runtime depending on the optional arguments passed into it.
```

File: docs/lang/articles/get-started/hello_world.md [hunk 4]
```diff
@@ -70,44 +71,57 @@ Save the code above to your local machine and run this program.
...
+import taichi as ti
+import taichi.math as tm
+
+ti.init(arch=ti.gpu)
+```
+
+### Define a Taichi field
+
+:::note
+`ti.field(x,y)` - Defines a Taichi field whose shape is of `y` and whose elements are of type `x`.
```
Comment (neozhaoliang) [inline docs/lang/articles/get-started/hello_world.md:114]:
```suggestion
`ti.field(dtype, shape)` - Defines a Taichi field whose shape is of `shape` and whose elements are of type `dtype`.
```

File: docs/lang/articles/get-started/hello_world.md [hunk 5]
```diff
@@ -70,44 +71,57 @@ Save the code above to your local machine and run this program.
...
 
 You can also directly specify which GPU backend to use. For example, set `arch=ti.cuda` to run your program on CUDA. Taichi raises an error if the target architecture is unavailable. See the [Global Settings](../reference/global_settings.md) for more information about `ti.init()`.
 
-### Define a field
+```python{4}
+import taichi as ti
+import taichi.math as tm
+
+ti.init(arch=ti.gpu)
+```
```
Comment (neozhaoliang) [inline docs/lang/articles/get-started/hello_world.md:109]:
```suggestion
```python{4}
import taichi as ti
import taichi.math as tm

ti.init(arch=ti.gpu)
```
A question here: is it intended to move all three lines at the end of the section `Initialize Taichi`? 
```

Comment (vai9er) [reply docs/lang/articles/get-started/hello_world.md:109]: Yes, in the first meeting we had we touched upon having the dialogue before the code snippet. If you'd like, I can add some extra dialogue to explain the code itself.

**Pull request merged**